test/test_torch.py

```diff
@@ -2480,6 +2480,21 @@ def test_numel(self):
         self.assertEqual(b.nelement(), 3 * 100 * 100)
         self.assertEqual(b.numel(), 3 * 100 * 100)
 
+    def test_nbytes(self):
+        self.assertEqual(torch.randn(3, dtype=torch.float32).nbytes, 12)
+        self.assertEqual(torch.randn(3, dtype=torch.float64).nbytes, 24)
```

Comment:
can we add a test where you the size of the storage is different than the size of the tensor?

e.g. something like:
`self.assertEqual(torch.randn(6, dtype=torch.float32)[::2].nbytes, 12)`.